### PR TITLE
SQL-1353: Remove custom query from connection dialogue make available as shared function

### PR DESCRIPTION
**Pull request by bucaojit**
Connection Dialogue is now only the URI and Database fields.
Users can run the Query function through the Advanced editor during the `Source` step.
```
= MongoDBAtlasODBC.Query("mongodb://localhost", "integration_test", "select * from example", [])
```

Try testing the new changes with Power BI and confirm queries work, and on-premise gateway if it's still setup.

### CODE REVIEW

File: connector/MongoDBAtlasODBC.pq [hunk 1]
```diff
@@ -172,8 +205,7 @@ MongoDBAtlasODBC = [
             {"MongoDBAtlasODBC.Contents", mongodbUri, database},
     Authentication = [
         UsernamePassword = []
-    ],
-    SupportsEncryption = true
```
Comment (bucaojit) [inline connector/MongoDBAtlasODBC.pq:176]: This change was suggested by Matt from Microsoft, it's not needed.

**Comment by terakilobyte** [comment]
Changes look good, but I wonder if we _must_ remove the query part in the connection window. The new flow is an inferior experience, imo. It's not fully intuitive, and requires a lot more work on the user's part to both change the query and then delete the navigation stage 😢 .

**Review by jchemburkar — APPROVED** [review]
lgtm!

**Comment by bucaojit** [comment]
evergreen merge

**Pull request merged**